package.json

```diff
@@ -18,19 +18,65 @@
             "type": "object",
             "title": "Doxygen Documentation Generator Settings",
             "properties": {
-                "doxdocgen.generic.commentStart": {
-                    "description": "Doxygen comment start indicator. Default ist /** but Qt style with /*! is also valid.",
+                "doxdocgen.generic.triggerSequence": {
+                    "description": "Doxygen comment trigger. This character sequence triggers generation of DoxyGen comments.",
                     "type": "string",
-                    "default": "/**",
-                    "enum": [
-                        "/**",
-                        "/*!"
-                    ]
+                    "default": "/**"
                 },
-                "doxdocgen.generic.generateReturnType": {
-                    "description": "Insert return type into generated documentation",
+                "doxdocgen.generic.firstLine": {
+                    "description": "The first line of the comment that gets generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "/**"
+                },
+                "doxdocgen.generic.commentPrefix": {
+                    "description": "The pre fix that is used for each comment line.",
+                    "type": "string",
+                    "default": " * "
+                },
+                "doxdocgen.generic.lastLine": {
```

Comment:
Does this make sense?
I mean being able to generate incomplete comments seems to be strange.

Reply:
/** is not the only DoxyGen style comments. At my work we use ///. Having a empty /// at the end makes no sense for that. Simiiar if you xml style params this would also make sense.

Reply:
Yes indeed. I see where you're getting at.

Thanks for sharing this.